src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,4 +1,11 @@
-<div class="panel panel-default" ng-switch on="panel.type" ng-if="savedObj || error">
+<div
+  class="panel panel-default"
+  ng-switch on="panel.type"
+  ng-if="savedObj || error"
+  reporting-item
+  reporting-item-title="{{savedObj.title}}"
+  reporting-item-description="{{savedObj.description}}"
```

Comment:
`object-`?  We call them saved objects all over the place, so that seems relatively appropriate.